sentry_sdk/integrations/__init__.py

```diff
@@ -124,7 +125,7 @@ def setup_integrations(
     with_auto_enabling_integrations=False,
     disabled_integrations=None,
 ):
-    # type: (Sequence[Integration], bool, bool, Optional[Sequence[Integration]]) -> Dict[str, Integration]
+    # type: (Sequence[Integration], bool, bool, Optional[Sequence[Union[type, Integration]]]) -> Dict[str, Integration]
```

Comment:
I think we probably only want to accept `Integration` types here, which we can indicate with `type[Integration]`.

```suggestion
    # type: (Sequence[Integration], bool, bool, Optional[Sequence[Union[type[Integration], Integration]]]) -> Dict[str, Integration]
```